inference-engine/src/cldnn_engine/CMakeLists.txt

```diff
@@ -25,11 +25,7 @@ target_link_libraries(${TARGET_NAME} PRIVATE clDNN_lib pugixml
                                              inference_engine inference_engine_legacy
                                              inference_engine_transformations)
 
-if(USE_CNNNETWORK_LPT)
```

Comment:
What's about `USE_CNNNETWORK_LPT` usages in the plugins code?

Reply:
done, thanks!